index.html

```diff
@@ -2,401 +2,285 @@
 <html lang="en">
 
 <head>
-  <title> DavidI Igbo | Portfolio </title>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" media="(min-width:451px)" href="tablet.css">
-  <link rel="stylesheet" href="style.css">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" type="text/css" />
+  <title>David Igbo | Portfolio</title>
 </head>
 
 <body>
-  <div class="section-1">
-    <header>
-      <div class="logo">
-        <a href="#" class="logo">My Logo</a>
-      </div>
-      <nav>
-        <div class="icon-div desktop-hidden">
-          <a href="javascript:void(0);" class="icon desktop-hidden">
-            <i class="fa fa-bars"></i>
-          </a>
+  <section class="wrapper">
+    <header class="menu-wrap">
+      <nav class="logo">
+        <a href="#" class="brand">David Igbo</a>
+        <input type="checkbox" class="toggler" id="toggle" onclick="toggle()" />
+        <div class="hamburger">
+          <div></div>
         </div>
-        <div class="nav-links mobile-hidden">
-          <ol>
-            <li><a href="#" class="nav-link">Portfolio</a></li>
-            <li><a href="#" class="nav-link">About</a></li>
-            <li><a href="#" class="nav-link">Contact</a></li>
-          </ol>
+        <div class="menu">
+          <div class="overlay">
+            <div>
+              <ul>
+                <li><a href="#works" class="nav-links">Portfolio</a></li>
+                <li><a href="#about" class="nav-links">About</a></li>
+                <li><a href="#contact" class="nav-links">Contact</a></li>
+              </ul>
+            </div>
+          </div>
```

Comment:
- [ ] Hello @david30-maker, well done working on some of the changes suggested by the previous reviewer, but currently i notice the blurring of the main page when the mobile menu is displayed functionality suggested by the previous reviewer is not implemented yet. In order to achieve this, you can simply target the container holding the mobile menu and give it a CSS property of `backdrop-filter: blur(10px)`. Kindly work on this before you resubmit for review.

Your design 👇 

![dave](https://user-images.githubusercontent.com/78634841/217340795-901c7686-5de2-439f-8e53-21c1391b1016.png)

Reply:
@brainconnect93, please I have tried my best to make background blurry but it is working for me, can you help me with it? Thanks